dbt_subprojects/tokens/models/transfers_and_balances/bitcoin/tokens_bitcoin_net_transfers_daily_asset.sql

```diff
@@ -0,0 +1,15 @@
+{{ config(
+        schema = 'tokens_bitcoin'
+        , alias = 'net_transfers_daily_asset'
+        , materialized = 'view'
+)
+}}
+
+select
+    blockchain
+    , block_date
+    , cast('0x0000000000000000000000000000000000' as varchar(42)) as contract_address
```

Comment:
```suggestion
    , '0000000000000000000000000000000000' as contract_address
```

Reply:
we should be able to just wrap in single quotes and not cast. also, according to linear, we are using this dummy value rather than varbinary burn address format